metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          "Apache 2.0"
 description      "Installs/Configures Sensu"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.5.0"
+version          "2.5.4"
```

Comment:
Could you please revert this line change. The version will be bumped accordingly when a release is cut.